tests/setup.js

```diff
@@ -0,0 +1,55 @@
+import { readdirSync, readFileSync } from "node:fs";
+import path from "node:path";
+import { JSDOM } from "jsdom";
+import { fileURLToPath } from "url";
+import { execa } from "execa";
+
+const __dirname = fileURLToPath(new URL(".", import.meta.url));
+export async function getIndexHtml(scenario) {
```

Comment:
Let's add a blank line for readability.

```suggestion

export async function getIndexHtml(scenario) {
```